### ISSUE TITLE
capture deployment details

### ISSUE THREAD

**Issue by piyush94** [DeployHubProject/DeployHub-Pro#328]
Hi @sbtaylor15, is it currently possible to capture deployment details for files (executables, tars, zips, iso, etc.)?

**Comment by sbtaylor15** [comment]
@piyush94 do you want to store the artifacts in the database like we did with the Helm charts?

**Comment by piyush94** [comment]
@sbtaylor15 no, only deployment details are fine as the artifact size can be in GBs.

**Comment by sbtaylor15** [comment]
@piyush94 go into a deployment log for a "files" deployment and then choose the Files tab.  Let us know if the deployed files are in the table or not.

**Comment by piyush94** [comment]
@sbtaylor15 yes, deployed files do show up the table.

**Comment by sbtaylor15** [comment]
@piyush94 what format do you need the captured data in?  via the API like the helm capture export?

**Comment by piyush94** [comment]
@sbtaylor15 JSON format is good via the API.
